glib-build-tools/README.md

```diff
@@ -0,0 +1,41 @@
+# GIO build helpers
+
+Crate containing helpers for building GIO-based applications.
+
+## Minimum supported Rust version
+
+Currently, the minimum supported Rust version is `1.57.0`.
+
+## Documentation
+
+* [Stable API](https://gtk-rs.org/gtk-rs-core/stable/latest/docs/glib-build-tools)
+* [Development API](https://gtk-rs.org/gtk-rs-core/git/latest/docs/glib-build-tools)
+
+## Using
+
+We recommend using [crates from crates.io](https://crates.io/keywords/gtk-rs) like this:
```

Comment:
Is any of this relevant to this crate? I don't think so as It doesn't link to glib/gio at all

Reply:
fair point, I simply used an existing readme as template, should I just remove the entire section then?

Reply:
I've removed it